index.html

```diff
@@ -1050,22 +1050,20 @@ <h3>Data delivery</h3>
               {{PressureRecord/[[State]]}} set to |state|
               and {{PressureRecord/[[Time]]}} set to |timestamp|.
             </li>
-            <li>
-              Increase |observer|.{{PressureObserver/[[ChangesCountMap]]}}[|source|].
-            </li>
             <li>
               If running [=passes rate obfuscation test=] with |observer| and |source| returns false:
               <ol>
                 <li>
-                  If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists,
-                  set observer.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] to |record|
-                  and [=iteration/continue=].
-                </li>
+                  Let |replacingRecord| be true if |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists
+                  or false otherwise.
                 <li>
                   Set |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|] to |record|.
                 </li>
```

Comment:
Here we should add:
Set |observer|.{{PressureObserver/[[LastRecordMap]]}}[|source|] to |record|.

This would prevent to have an extra update passing "has change in data test" on the next round.

Reply:
Sounds good to me. I was about to say, just set it before then :)

Reply:
The problem is a bit more complex than that, not difficut, but few corner cases. I m rewriting  a proposal, I hope @kenchris and @rakuco can go toroughtly through it.

Reply:
It should be now Final and working. Unless you see some more issues.